cEP-0005.md

```diff
@@ -104,7 +104,7 @@ class RedundancyBear(LocalBear):
     # ... and detect code clones.
     DETECT_ASPECTS = {Root.Redundancy.Clone}
 
-    # Aspects are passed as parameter
+    # aspects are passed as parameter
```

Comment:
You should somehow make clear that we pass aspect instances here in contrast to the aspect classes that are passed to the bear class definition